src/views/Home.vue

```diff
@@ -1,26 +1,27 @@
 <template>
   <v-container class="d-flex flex-column align-center">
     <PageViewer :pageTitle="pageTitle" :content="content" />
-
   </v-container>
-
 </template>
 
-<script>
-import { ref } from "vue";
+<script setup>
+import { ref, onMounted, inject } from "vue";
 import PageViewer from "../components/PageViewer.vue";
-export default {
-  components: {
-    PageViewer,
-  },
-  setup() {
-    const pageTitle = ref("Home Page");
-    const content = ref("This is the home content");
+import { useStore } from "vuex";
+const store = useStore();
+const pageTitle = ref("Home Page");
+const content = ref("This is the home content");
+const snackbarProvider = inject("snackbarProvider");
 
-    return {
-      pageTitle,
-      content,
-    };
-  },
-};
-</script>
\ No newline at end of file
+onMounted(async () => {
```

Comment:
Using onMounted here might be fine but the method is executed after the current DOM tree has been generated, meaning if any of the components defined in Home needs any of the store values, then they would fail. We might consider using onBeforeMounted here? https://vuejs.org/api/composition-api-lifecycle.html#onmounted

Reply:
should be better, yes. Changed

Reply:
It's still onMounted. Should it be changed?